workers/unity/Packages/io.improbable.gdk.core/.codegen/Source/Generators/Core/UnityComponentDataGenerator.cs

```diff
@@ -1,18 +1,20 @@
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Runtime.InteropServices;
 using Improbable.Gdk.CodeGeneration.CodeWriter;
 using Improbable.Gdk.CodeGeneration.CodeWriter.Scopes;
+using Improbable.Gdk.CodeGeneration.Jobs;
```

Comment:
What is this imported for?

Reply:
this + `System` are not needed and have been removed in a future commit